products/kotlin.md

```diff
@@ -24,8 +24,8 @@ auto:
 releases:
 -   releaseCycle: "1.8"
     eol: false
-    latest: "1.8.0"
-    latestReleaseDate: 2022-12-28
+    latest: "1.8.10"
```

Comment:
Odd, but considering [1.8.9 does not exist](https://github.com/JetBrains/kotlin/releases/tag/v1.8.9) LGTM.